debian/stretch/foreman/foreman.postinst

```diff
@@ -101,6 +101,10 @@ fi
 # Own all the core files
 chown -Rf foreman:foreman '/usr/share/foreman'
 
+# During dh_install we cannot rename files, so we do so here
```

Comment:
nope, please never rename files in postinst, esp not the ones in /etc as those are handled as conffiles